### ISSUE TITLE
12.1.1  - HIgh Level Security

### ISSUE THREAD

**Issue by mcr** [ietf-rats-wg/architecture#220]
````
 For high level security, specialized hardware will likely be used
   providing protection against chip decapping attacks, power supply and
   clock glitching, faulting injection and RF and power side channel
   attacks.
````

This section, as put, is kind of a big leap, no?
I think the important part of this next level, is the introduction of a hardware root of trust like a TPM to *eliminate* the need to trust the device’s operating system software in reporting attestation results.

**Comment by laurencelundblade** [comment]
@laurencelundblade 

How about "For strongest security,...." to avoid confusion with high level OS.

**Comment by mcr** [comment]
This also already fixed by existing text #282.
